nominal/core/client.py

```diff
@@ -726,7 +742,8 @@ def get_channel(self, rid: str) -> Channel:
                 This typically occurs when there is no such channel for the given RID.
         """
         return Channel._from_conjure_logicalseries_api(
-            self._clients, self._clients.logical_series.get_logical_series(self._clients.auth_header, rid)
+            self._clients,
+            self._clients.logical_series.get_logical_series(self._clients.auth_header, rid),
```

Comment:
my guess is that you have `black` set to autoformat rather than `ruff` and hence isn't picking up the `ruff` options here and formatting differently? should make sure the PR isn't changing other formatting stuff as it gives greater chance for conflicts.